crates/objc2/src/macros/mod.rs

```diff
@@ -258,6 +263,21 @@ macro_rules! sel {
     };
 }
 
+#[macro_export]
+#[cfg(all(
+    not(doc),
+    feature = "unstable-core-ffi-objc",
+    not(feature = "unstable-static-sel")
+))]
+macro_rules! sel {
```

Comment:
Nit: Could you overwrite `__sel_inner` instead? That way, the docs on `sel!` are still available.

Reply:
It'd also be nice to have a section explaining the `unstable-core-ffi-objc` feature flag in the `sel!` macro's `# Features` section.